PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+Related issue:
```

Comment:
I have no idea why this file keeps popping in and out of existence between branches. It's maddening.